pkg/test/resourcefixture/testdata/basic/compute/v1alpha1/computemanagedsslcertificate/managedsslcertificate/create.yaml

```diff
@@ -11,15 +11,14 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
 apiVersion: compute.cnrm.cloud.google.com/v1alpha1
 kind: ComputeManagedSSLCertificate
 metadata:
-  name: computemanagedsslcertificate-${uniqueId}
+  name: computemanagedsslcertificate${uniqueId}
```

Comment:
nit: I think the [naming convention](https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/README.NewResourceFromTerraform.md#naming-conventions-special-variables-and-constants-in-testdata) is  computemanagedsslcertificate-${uniqueId}.

Reply:
In this case, I think the dash violated a regex for the ssl cert name since we are using `metadata.name` for that BUT I am not able to reproduce that violation anymore so maybe I was getting confused 🤔